src/impl/formatters/ethAddressHtmlFormatter.ts

```diff
@@ -0,0 +1,18 @@
+import { Feature } from 'src/interfaces/feature';
+
+export class EthAddressHtmlFormatter implements Feature {
+    public readonly globalName: string = "http://types.dapplets.org/ethereum/formatters/address/1.0";
+    public readonly viewGlobalName: string = "http://types.dapplets.org/views/htmlMustache/1.0";
```

Comment:
rename `viewGlobalName` => `gn_targetView` ? 
"gn_" - prefix for a global name...

Reply:
Looks like `globalName` and `viewGlobalName` are private properties because it isn't used by external classes and used only for `regKey` concatenation:
`public readonly regKey = [this.globalName, this.viewGlobalName];`

And if it's private properties, I think there is no difference in how to name this :)

Reply:
Also, I can't find any examples of using a such convention about `gn_` prefix.